.eslintrc

```diff
@@ -34,5 +34,6 @@
   ],
   "globals": {
     "gapi": false,
+    "google": false,
```

Comment:
rather than making this global across all the files in the repo, I would restrict it to just the file(s) that use the global. You can do this by adding a comment at the top of the file that looks like this:

```jsx
/* globals google */
```